examples/highavailability_mode/README.md

```diff
@@ -0,0 +1,6 @@
+# High-availability mode Client to Site VPN example
+
+An end-to-end example that creates the client to site VPN in the high availability mode.
+This example uses the IBM Cloud terraform provider to:
+ - Create a new resource group if one is not passed in.
+ - Create a new VPC in the resource group and region provided.
```

Comment:
We do create a bunch of other stuff as well in the example such as SM, Certs etc.

Reply:
I think this should be tackled separately - content here is taken from active module internally without content changes.